[terra-status-view] - Update Doc example (#3810)
cerner/terra-core: packages/terra-core-docs/CHANGELOG.md

## Unreleased

* Changed

>  * Updated Custom Icon + message + buttons example for `terra-status-view`.

Reviewer: ```suggestion
 * Updated `Custom Icon + message + buttons` example for `terra-status-view`.
```

Author: Changes Made.

>  * Updated `terra-status-view` Change Variant example component from HTML Native select to `terra-form-select` component.

Reviewer: ```suggestion
 * Updated `terra-status-view - Change Variant` example component to use the `terra-form-select` component instead of HTML native `<select>`.
```

Author: changes resolved.
